File: src/Api/Model/DB/PorteEmpresa.cs
using Realms;
using System;
using System.Collections.Generic;
using System.Text;

namespace CriadorBaseDados.Model.DB
{
    public class PorteEmpresa : RealmObject
    {
        [PrimaryKey]
        public int Cod { get; set; }
        public string Descricao { get; set; }
    }
}

File: src/Mobile/b2b/lib/model/core/porte_empresa.dart
class PorteEmpresa {
  final int cod;
  final String descricao;

  PorteEmpresa(this.cod, this.descricao);

  PorteEmpresa.fromJson(Map<String, dynamic> json)
      : cod = json['cod'],
        descricao = json['descricao'];

  Map<String, dynamic> toMap() => <String, dynamic>{
        'cod': cod,
        'descricao': descricao,
      };
}
